pyproject.toml

```diff
@@ -81,33 +80,16 @@ include = [
     "spaghetti.*",
 ]
 
-[tool.black]
-line-length = 88
-extend-exclude = '''
-(
-  docs/conf.py
-)
-#'''
-
 [tool.ruff]
 line-length = 88
 select = ["E", "F", "W", "I", "UP", "N", "B", "A", "C4", "SIM", "ARG"]
-target-version = "py38"
-ignore = [
-    "B006",
-    "B008",
-    "B009",
-    "B010",
-    "C408",
-    "E731",
-    "F401",
-    "F403",
-    "N803",
-    "N806",
-    "N999",
-    "UP007"
-]
+target-version = "py310"
```

Comment:
I suggest removing this as Ruff is able to deduce it from `requires-python` and it will be one place less to worry about when bumping versions.

Reply:
Ah, yeah. Good point.